_common/core_ref.md

```diff
@@ -1,5 +1,18 @@
 ---
-layout: externalurl
-redirect_url: https://core.blockstack.org/
+layout: core
 title: "Stacks Node API"
+description: Interacting with the Blockstack Blockchain
+permalink: /:collection/:path.html
 ---
+
+With the launch of Stacks 2.0, a new version of the Blockstack blockchain was released. There are two ways of interacting with the blockchain, either using a RESTful JSON API or by making JSON RPC calls to the blockchain core node.
+
+## JSON RPC API
+The Stacks 2.0 blockchain's Rust implementation exposes JSON RPC endpoints, which can be used to interface with blockchain. [You can find the RPC API references here](https://github.com/blockstack/stacks-blockchain/blob/master/docs/rpc-endpoints.md).
+
+## RESTful JSON API
+A RESTful JSON API that is hosted by PBC and has a proxy to the JSON RPC endpoints. [You can find the OpenAPI specification and documentation here](https://blockstack.github.io/stacks-blockchain-sidecar/). You can also make API calls from within the documentation.
```

Comment:
```suggestion
A RESTful JSON API that is hosted by PBC and has a proxy to the JSON RPC endpoints. [You can find the OpenAPI specification and documentation here](https://blockstack.github.io/stacks-blockchain-sidecar/).
```
Redoc doesn't support this out the box. Maybe something we can add later, for the time being we can instruct to use `curl` or `httpie`